### PR TITLE
Update about_Scheduled_Jobs_Troubleshooting.md

### PR DESCRIPTION
**Pull request by PrzemyslawKlys**
Links added, code highlighted

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by PrzemyslawKlys** [comment]
Updated links, some code highlights

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psscheduledjob/about/about_scheduled_jobs_troubleshooting?view=powershell-5.1&branch=pr-en-us-2717) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md)

<a name="reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md"></a>
### [reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807211653312319-2717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FDX1u3bs3hBl%2Fxi09npIlSLKngx1lswe9%2BkaDOsdUCA%3D&st=2018-07-21T17%3A02%3A51Z&se=2018-08-21T17%3A07%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
Thanks for the contribution but I cannot accept this as submitted. Please see https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/3-STYLE-GUIDE.md#about_-file-formatting. 

About_* files have very specific formatting requirements. The lines must be wrapped at 80 columns. Code blocks and tables must be wrapped at 76 columns. Please do not change the line wrappings.

**Comment by sdwheeler** [comment]
@PrzemyslawKlys  If you can fix the line wraps I will reconsider this PR.
Thanks

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 1]
```diff
@@ -17,23 +17,19 @@ Explains how to resolve problems with scheduled jobs
 
 # LONG DESCRIPTION
 
-This section describes some of the problems that you might encounter when
-using the scheduled job features of Windows PowerShell and it suggests
-solutions to these problems.
+This section describes some of the problems that you might encounter when using the scheduled job features of Windows PowerShell and it suggests solutions to these problems.
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Do not change the line breaks. About topics have specific formatting requirements.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psscheduledjob/about/about_scheduled_jobs_troubleshooting?view=powershell-5.1&branch=pr-en-us-2717) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md)

<a name="reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md"></a>
### [reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807231449152864-2717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sdnRz1ApqU1HWeZpA9em2VP%2F8dTFmFABsoEnoU4JlV8%3D&st=2018-07-23T14%3A57%3A51Z&se=2018-08-23T15%3A02%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by PrzemyslawKlys** [comment]
Can you verify now?

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 2]
```diff
@@ -77,13 +80,15 @@ ProcessJob 48 11/7/2011 12:00:01 AM
 ProcessJob 49 11/7/2011 3:00:02 AM
 ProcessJob 50 11/8/2011 3:00:02 AM
 
+```
+
 To get the results of an instance of a scheduled job, use the
 Receive-Job cmdlet. The following command gets the results of the
 newest instance of the ProcessJob (ID = 50).
-
+```
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Need a blank line before and after the code fence.

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 3]
```diff
@@ -184,12 +202,14 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id ProcessName
 1213      33    12348      21676    88    25.71   1608 CcmExec
 29       4     1168       2920    43     0.02    748 conhost
 46       6     2208       4612    45     0.03   1640 conhost
-# ...
 
+```
+
+# ...
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: This is supposed to be inside the code fence to indicate more output that was truncated from the example.

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 4]
```diff
@@ -206,18 +226,22 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id ProcessName
 1213      33    12348      21676    88    25.71   1608 CcmExec
 29       4     1168       2920    43     0.02    748 conhost
 46       6     2208       4612    45     0.03   1640 conhost
-
--- The scheduled job might be corrupted.
+```
+# The scheduled job might be corrupted.
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Need a blank line after the code fence.

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 5]
```diff
@@ -184,12 +202,14 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id ProcessName
...
-# ...
 
+```
+
+# ...
 
 To get the result of a job instance more than one time in a session,
 use the Keep parameter of the Receive-Job cmdlet.
-
+```
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Need a blank line before the code fence.

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 6]
```diff
@@ -406,22 +459,21 @@ error message each time you run the Get-ScheduledJob cmdlet.
...
 
-The directory is located at $env:UserProfile\AppData\Local\Microsoft\Windows\PowerShell
-\ScheduledJobs<ScheduledJobName>
+The directory is located at `$env:UserProfile\AppData\Local\Microsoft\Windows\PowerShell
+\ScheduledJobs<ScheduledJobName>`
 
 Typically:
-C:\Users<UserName>\AppData\Local\Microsoft\Windows\PowerShell
-\ScheduledJobs<ScheduledJobName>.
+`C:\Users<UserName>\AppData\Local\Microsoft\Windows\PowerShell\ScheduledJobs<ScheduledJobName>.`
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Restore the line break from the original formatting.

File: reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md [hunk 7]
```diff
@@ -368,18 +418,22 @@ Windows PowerShell.
...
 Use the following methods to detect and correct job failures.
 
--- Check the Task Scheduler event log for errors. To check the log,
+## Check the Task Scheduler event log for errors. To check the log,
 use Event Viewer or a Windows PowerShell command such as the
 following:
 
+```
+
 Get-WinEvent -LogName Microsoft-Windows-TaskScheduler/Operational | Where {$_.Message -like "fail"}
```
Comment (sdwheeler) [inline reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md]: Change this to: 

```powershell
Get-WinEvent -LogName Microsoft-Windows-TaskScheduler/Operational |
  Where {$_.Message -like "fail"} 
```

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psscheduledjob/about/about_scheduled_jobs_troubleshooting?view=powershell-5.1&branch=pr-en-us-2717) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md)

<a name="reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md"></a>
### [reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807231532314796-2717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=n1m4ZHy3OfBim1yWyLlDDRpbt%2BzdnaCWMpeBYV2aiuU%3D&st=2018-07-23T15%3A41%3A25Z&se=2018-08-23T15%3A46%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/psscheduledjob/about/about_scheduled_jobs_troubleshooting?view=powershell-5.1&branch=pr-en-us-2717) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md)

<a name="reference%2F5.1%2FPSScheduledJob%2FAbout%2Fabout_Scheduled_Jobs_Troubleshooting.md"></a>
### [reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md](https://github.com/PrzemyslawKlys/PowerShell-Docs/blob/patch-2/reference/5.1/PSScheduledJob/About/about_Scheduled_Jobs_Troubleshooting.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201807231551164365-2717%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fk5scxEskS5Yijvnp4bHCBcFiOJ10YrJzGJu44xE3eg%3D&st=2018-07-23T16%3A00%3A31Z&se=2018-08-23T16%3A05%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**